src/docker-outside-of-docker/install.sh

```diff
@@ -99,6 +99,22 @@ find_version_from_git_tags() {
     echo "${variable_name}=${!variable_name}"
 }
 
+# Function to fetch the previous version of the plugin
+get_previous_version() {
+    repo_url=$1
+    # this would del the assets key and then get the second encountered tag_name's value from the filtered array of objects
+    curl -s "$repo_url" | jq -r 'del(.[].assets) | .[1].tag_name' 
```

Comment:
```suggestion
    curl -s "$repo_url" | jq -r 'del(.[].assets) | .[0].tag_name' 
```

Shouldn't this be updated to ^? 

![image](https://github.com/devcontainers/features/assets/24955913/980c58bc-9581-4e15-8a1d-9f0f09f6d8a2)

^ However, the latest available release is v1.0.5, see https://github.com/docker/compose-switch/releases/tag/v1.0.5

Reply:
Can you update this in other Features as well where you previously made these changes?

Reply:
reverted to this logic